R/D1Client.R

```diff
@@ -402,7 +402,7 @@ setMethod("getDataObject", "D1Client", function(x, identifier, lazyLoad=FALSE, l
         }
       } 
     }
-  }
+  } else currentMN <- x@mn
```

Comment:
```suggestion
  } else {
      currentMN <- x@mn
  }
```

if else blocks should include braces to avoid potential bugs.